src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--firstDay--wed calendar--amountOfDays--29">
+      <div class="calendar__square-1"></div>
+      <div class="calendar__square-2"></div>
+      <div class="calendar__square-3"></div>
+      <div class="calendar__square-4"></div>
+      <div class="calendar__square-5"></div>
+      <div class="calendar__square-6"></div>
+      <div class="calendar__square-7"></div>
+      <div class="calendar__square-8"></div>
+      <div class="calendar__square-9"></div>
+      <div class="calendar__square-10"></div>
+      <div class="calendar__square-11"></div>
+      <div class="calendar__square-12"></div>
+      <div class="calendar__square-13"></div>
+      <div class="calendar__square-14"></div>
+      <div class="calendar__square-15"></div>
+      <div class="calendar__square-16"></div>
+      <div class="calendar__square-17"></div>
+      <div class="calendar__square-18"></div>
+      <div class="calendar__square-19"></div>
+      <div class="calendar__square-20"></div>
+      <div class="calendar__square-21"></div>
+      <div class="calendar__square-22"></div>
+      <div class="calendar__square-23"></div>
+      <div class="calendar__square-24"></div>
+      <div class="calendar__square-25"></div>
+      <div class="calendar__square-26"></div>
+      <div class="calendar__square-27"></div>
+      <div class="calendar__square-28"></div>
+      <div class="calendar__square-29"></div>
+      <div class="calendar__square-30"></div>
+      <div class="calendar__square-31"></div>
```

Comment:
you can use only 1 common class and `nth-child`

Reply:
![image](https://user-images.githubusercontent.com/102521367/177035516-575886b1-21d9-40e6-9b7e-07dab357d183.png)
I edited, but BEM lint says: